packages/ui/src/components/va-select/VaSelect.vue

```diff
@@ -221,12 +221,8 @@ export default defineComponent({
     autoSelectFirstOption: { type: Boolean, default: false },
 
     // Input style
-    outline: { type: Boolean, default: false },
-    bordered: { type: Boolean, default: false },
-    label: { type: String, default: '' },
     placeholder: { type: String, default: '' },
     searchPlaceholderText: { type: String, default: '$t:search' },
```

Comment:
![image](https://github.com/epicmaxco/vuestic-ui/assets/35342240/2c4e7b73-c06e-499b-9dcc-cef91c46f61a)
Why is the icon not visible here?